cmsis/TARGET_CORTEX_M/mbed_fault_handler.c

```diff
@@ -69,24 +75,24 @@ void mbed_fault_handler (uint32_t fault_type, void *mbed_fault_context_in)
     mbed_error_printf("\n\n-- MbedOS Fault Handler --\n\n");
     
     //Now call mbed_error, to log the error and halt the system
-    mbed_error( faultStatus, "Fault exception", mbed_fault_context.PC_reg, NULL, 0 );
+    mbed_error( faultStatus, "Fault exception", mbed_fault_context->PC_reg, NULL, 0 );
     
 }
 
 MBED_NOINLINE void print_context_info(void)
 {
     //Context Regs
     for(int i=0;i<13;i++) {
-        mbed_error_printf("\nR%-4d: %08" PRIX32, i, ((uint32_t *)&mbed_fault_context)[i]);
+        mbed_error_printf("\nR%-4d: %08X", i, ((uint32_t *)(mbed_fault_context))[i]);  
     }
         
-    mbed_error_printf("\nSP   : %08" PRIX32
```

Comment:
Why are the printf format directives being removed? These were a fairly recent addition:

https://github.com/ARMmbed/mbed-os/pull/8601